src/core/distributions/phylogenetics/substitution/AbstractPhyloCTMCSiteHomogeneous.h

```diff
@@ -1535,7 +1503,7 @@ bool RevBayesCore::AbstractPhyloCTMCSiteHomogeneous<charType>::recursivelyDrawSt
     transition_states.push_back(start_state);
 
     // get the rate matrix for this branch (or site if using a mixture of matrices over sites)
-    RateMatrix_JC jc(this->num_chars);
+    RateMatrix_JC jc(this->num_chars); // BAD!!
```

Comment:
Yes, any better ideas?

Reply:
I just want to mark this to work on later.